pkg/cli/democluster/api/api.go

```diff
@@ -0,0 +1,49 @@
+// Copyright 2021 The Cockroach Authors.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+package democlusterapi
```

Comment:
should the package name match `api`?

Reply:
The package name is the prefix that's going to be used in calls by client packages.
If the reader sees `api.XXX` it does not say "which API it is".
With `democlusterapi.XXX` it's clearer where XXX comes from.

Meanwhile, I chose the `api` directory name because the immediate parent directory is `democluster` so it's kind of clear *when looking at the file system path* which API we're looking at.

We have many other packages where the package name does not match the import path - that's a standard Go feature.